### PR TITLE
Fix basic visibility and raster ITs

### PR DESCRIPTION
**Pull request by boh1018**
Fixed integration tests relating to visibility (set in `KuduOptions`), maximum cell size (64 Kb), and row deletion predicates

Signed-off-by: foolhb <hanbo1018@gmail.com>

### CODE REVIEW

File: test/src/main/java/org/locationtech/geowave/test/KuduStoreTestEnvironment.java [hunk 1]
```diff
@@ -69,4 +69,10 @@ protected void initOptions(StoreFactoryOptions options) {
     kuduOptions.setKuduMaster("127.0.0.1:7051");
   }
 
+  @Override
+  public int getMaxCellSize() {
```
Comment (ChesleyTan) [inline test/src/main/java/org/locationtech/geowave/test/KuduStoreTestEnvironment.java:73]: Should we also add this to the other data stores with noted limitations?

File: test/src/test/java/org/locationtech/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java [hunk 2]
```diff
@@ -93,9 +93,16 @@ public static void reportTest() {
...
-    final int tileSize = 64; // 256 fails on bigtable exceeding maximum
-    // size, 128 fails on DynamoDB exceeding
-    // maximum size
+    final int maxCellSize =
+        TestUtils.getTestEnvironment(dataStoreOptions.getType()).getMaxCellSize();
+    final int tileSize;
+    if (maxCellSize <= 64 * 1024) {
+      tileSize = 24;
+    } else {
+      tileSize = 64; // 256 fails on bigtable exceeding maximum size
```
Comment (ChesleyTan) [inline test/src/test/java/org/locationtech/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java:102]: We should probably scale the tile size proportional to the max cell size

File: extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/config/KuduOptions.java [hunk 3]
```diff
@@ -8,52 +8,28 @@
...
+public class KuduOptions extends BaseDataStoreOptions {
+  @Override
+  public boolean isServerSideLibraryEnabled() {
+    return false;
   }
 
-  public KuduOptions(final String geowaveNamespace) {
-    super(geowaveNamespace);
+  @Override
+  protected int defaultMaxRangeDecomposition() {
```
Comment (ChesleyTan) [inline extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/config/KuduOptions.java]: Could we actually remove these overrides? These were copied over from another datastore, but there's no evidence that we need them for Kudu yet

File: test/src/main/java/org/locationtech/geowave/test/KuduStoreTestEnvironment.java (hunk 1 above)
Comment (dannyqiu) [reply test/src/main/java/org/locationtech/geowave/test/KuduStoreTestEnvironment.java:73]: As of right now, the other datastore work with the original values. I think that can be an incremental change for the other datastores.

File: test/src/test/java/org/locationtech/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java (hunk 2 above)
Comment (dannyqiu) [reply test/src/test/java/org/locationtech/geowave/test/basic/GeoWaveBasicCustomCRSRasterIT.java:102]: I agree, though that will most likely take more time and require its own PR.

File: extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/config/KuduOptions.java (hunk 3 above)
Comment (dannyqiu) [reply extensions/datastores/kudu/src/main/java/org/locationtech/geowave/datastore/kudu/config/KuduOptions.java]: :+1:

**Review by ChesleyTan — APPROVED** [review]

**Pull request merged**